### PR TITLE
Added validations and some improvements to supervisor custom login component

### LINKED ISSUE #26: [Impl] Improve the examiner custom login component.
Let's add a prop to handle the `use biometric login instead`.
When we use the component we will pass a function to handle this. 

Change the input field line color to color used in the design.

### PR DESCRIPTION
**Pull request by thilinawaks1999**

### CODE REVIEW

File: src/components/examiner-custom-login.js
Comment (jananga99) [inline src/components/examiner-custom-login.js:1]: Isn't the examiner/supervisor login via email and password (not the index and password)?

Can we please check?
https://www.figma.com/file/ewqkiF5GG353Ae8aP2f3df/Scheduled-Page?type=design&node-id=38-4270&mode=design&t=4otjH8hLiEDokeyd-0

image

**Review by jananga99 — CHANGES_REQUESTED** [review]

File: src/components/examiner-custom-login.js
Comment (Malith-19) [inline src/components/examiner-custom-login.js:1]: @thilinawaks1999

Comment (thilinawaks1999) [reply src/components/examiner-custom-login.js:1]: will fix it

Comment (thilinawaks1999) [reply src/components/examiner-custom-login.js:1]: Done

**Review by Malith-19 — APPROVED** [review]

File: src/components/examiner-custom-login.js [hunk 1]
```diff
@@ -22,33 +23,61 @@ function ExaminerCustomLoginComponent() {
...
+      .min(8, "Password is too short - should be 8 chars minimum.")
+      .max(20, "Password is too long - should be 20 chars maximum.")
+      .matches(
+        /^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[!@#$%^&*])(?=.{8,})/,
+        "Must Contain 8 Characters, One Uppercase, One Lowercase, One Number and One Special Case Character"
+      ),
+  });
+
+  const onSubmit = (values) => {
+    loginExaminer(values);
```
Comment (jananga99) [inline src/components/examiner-custom-login.js]: @thilinawaks1999  @Malith-19 Can we change this as follows?
```
loginExaminer(values, true);
```
As the function definition in https://github.com/cse-mosip/Exam-Attendance-UI/blob/main/src/context/appContext.js , if we are using loginExaminer with email, we need to make withEmail=true.

**Review by jananga99 — CHANGES_REQUESTED** [review]

File: src/components/examiner-custom-login.js (hunk 1 above)
Comment (thilinawaks1999) [reply src/components/examiner-custom-login.js]: Done

**Review by jananga99 — APPROVED** [review]

**Pull request merged**